web/src/message_list_tooltips.ts

```diff
@@ -123,8 +123,23 @@ export function destroy_all_message_list_tooltips(): void {
     message_list_tippy_instances.clear();
 }
 
-function get_last_modified_time_string(last_edit_timestamp: number): string {
-    const last_modified_time = new Date(last_edit_timestamp * 1000);
+function get_last_modified_time_string(
+    last_edit_timestamp: number | undefined,
+    last_moved_timestamp: number | undefined,
+): string {
+    let latest_timestamp: number;
+    if (last_edit_timestamp === undefined) {
+        assert(last_moved_timestamp !== undefined);
+        latest_timestamp = last_moved_timestamp;
+    } else if (last_moved_timestamp === undefined) {
+        assert(last_edit_timestamp !== undefined);
+        latest_timestamp = last_edit_timestamp;
+    } else if (last_edit_timestamp >= last_moved_timestamp) {
+        latest_timestamp = last_edit_timestamp;
+    } else {
+        latest_timestamp = last_moved_timestamp;
+    }
```

Comment:
I think it might be better to just display the "Last edited" and "Last moved" lines separately, if both are present? 

It seems less confusing that way.

Reply:
Checked with @alya and she agrees that design change for the tooltips is probably a good idea.

Reply:
Done. See screenshots below in the main updates comment.